charts/misc-apps/values.yaml

```diff
@@ -13,7 +13,7 @@ downscaler:
   # downscaler.chart -- Chart
   chart: "kube-downscaler"
   # downscaler.targetRevision -- [kube-downscaler Helm chart](https://github.com/helm/charts/tree/master/incubator/kube-downscaler) version
-  targetRevision: "0.5.*"
+  targetRevision: "0.6.*"
```

Comment:
Version `20.10.0` of kubedownscaler image https://github.com/helm/charts/commit/fc37da771f6c3d405b0e2b3d39f3a0e5a1732c67#diff-7c6021564b8e31f7e48abc401a2a48fb56f5900e63e3b29438dce4219d9d89e2